copypasta/Cargo.toml

```diff
@@ -16,7 +16,7 @@ objc_id = "0.1"
 objc-foundation = "0.1"
 
 [target.'cfg(all(unix, not(any(target_os="macos", target_os="android", target_os="emscripten"))))'.dependencies]
-x11-clipboard = "0.3"
+x11-clipboard = "0.4"
```

Comment:
Would you mind changing this to `0.4.0`? You didn't introduce this, but it's the idiomatic format so if we change it anyways, might as well improve this.

Reply:
All done.